### DISCUSSION TITLE
SUPIR Upscaler

### DISCUSSION THREAD

**Discussion by D0n-A** [vladmandic/sdnext#3114] [General]
Is it ever planned to support [this ](https://github.com/Fanghua-Yu/SUPIR)upscaler, so to speak, "out of the box"?

**Comment by vladmandic** [comment]
no, requirements that SUPIR have are just too great to have it as out-of-the-box feature.
for example: [LLaVA CLIP](https://huggingface.co/openai/clip-vit-large-patch14-336) and [LLaVA v1.5 13B](https://huggingface.co/liuhaotian/llava-v1.5-13b)

**Reply by SyamsQ** [reply]
how to install supir on SD.next?
